go/interpreter/lfvm/instructions.go

```diff
@@ -886,78 +886,16 @@ func checkInitCodeSize(c *context, size *uint256.Int) bool {
 }
 
 func opCreate(c *context) {
```

Comment:
nit, just like in push, swap, log. The call to genericCreate could be done directly from the interpreter switch.

Reply:
unless it would bring a big performance performance, I wouldn't do that. maybe in another PR we add a benchmarka and  do the change you suggest.